dsl/dsl.cc

```diff
@@ -161,6 +162,7 @@ unique_ptr<ast::Expression> DSL::run(core::MutableContext ctx, unique_ptr<ast::E
     DSLReplacer dslReplacer;
     ast = ast::TreeMap::apply(ctx, dslReplacer, std::move(ast));
     auto verifiedResult = ast::Verifier::run(ctx, std::move(ast));
+    verifiedResult = Flatten::patchFile(ctx, std::move(verifiedResult));
```

Comment:
can we use the same naming convention, ie. `run`?
why does it run after verifier?

Reply:
None of those were motivated decisions. I've changed it to `run` and put it before the verifier.